packages/font-icons-mdl2/src/index.ts

```diff
@@ -27,15 +27,15 @@ declare global {
   // eslint-disable-next-line @typescript-eslint/naming-convention
   interface Window {
     FabricConfig?: {
-      fontBaseUrl?: string;
+      iconBaseUrl?: string;
```

Comment:
This would be a breaking change. Instead, deprecate the old but support it. And add iconBaseUrl.

Reply:
Please add a comment here to explain why this exists. (In order to direct the base icon url prior to javascript loading, the config can be burned on the page.) Otherwise it won't be understood and future maintainers will be left guessing :)